src/components/Connect/Burner/ImportSeed/index.stories.jsx

```diff
@@ -1,10 +1,10 @@
 import theme from '../../../theme'
 import ThemeProvider from '../../../ThemeProvider'
-import InputMnemonic from './'
+import { ImportSeed } from '.'
```

Comment:
Default export naming here as well. I guess a seed is the same as Mnemonic, but still bad consistency